docs/sidecar-configuration-format.md

```diff
@@ -31,14 +31,30 @@ volumes:
 - name: nginx-conf
   configMap:
     name: nginx-configmap
+- name: some-config
+  configMap:
+    name: some-configmap
+
+# hostAliases are not being merged, only added, as they only add entries to /etc/hosts in the containers.
+# Duplicate entries won't throw an error.
+# hostAliases are used for the whole pod.
+hostAliases:
+  - ip: 1.2.3.4
+    hostnames:
+      - somehost.example.com
+      - anotherhost.example.com
 
 # all environment variables defined here will be added to containers _only_ if the .Name
 # is not already present (we will not replace an env var, only add them)
 # These will be inserted into each container in the pod, including any containers added via
-# injection.
+# injection. The same applies to volumeMounts.
 env:
 - name: DATACENTER
   value: "dc01"
+
```

Comment:
could you add a comment here clarifying behavior of this field, similar to the others?

Reply:
added some lines of documentation regarding volumeMounts.